incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -41,22 +69,19 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(hasSecret(ctx, fooName, "foo-sec")).Should(BeTrue())
 		// "foo-sec" is not propagated to bar because Secret hasn't been configured in HNCConfiguration.
 		Eventually(hasSecret(ctx, barName, "foo-sec")).Should(BeFalse())
-
-		Eventually(func() error {
-			c := getHNCConfig(ctx)
-			return addSecretToHNCConfig(ctx, c)
```

Comment:
Do we not need a test for this anymore? E.g. that a type that _is_ explicitly added to the config _should_ be propagated? It can be its own test now (eg we don't need a "before-and-after" test, we can have two tests, one that adds Secrets to the config and one that doesn't).

Reply:
I changed the test to option 2 that we discussed in the previous PR (https://github.com/kubernetes-sigs/multi-tenancy/pull/438#discussion_r381444776). I create objects of two types in the parent namespace and only add one type to config. The test shows that only objects of the type that is added to the config are populated to the child namespace. Feel free to take a look.